doc/api/buffer.md

```diff
@@ -1150,6 +1150,34 @@ console.log(utf16Buffer.indexOf('\u03a3', 0, 'ucs2'));
 console.log(utf16Buffer.indexOf('\u03a3', -4, 'ucs2'));
 ```
 
+Edge cases:
```

Comment:
No reason to say "Edge cases." Just start right in with "If `value is not a string..."